dom5.ts

```diff
@@ -38,6 +38,15 @@ export function hasAttribute(element: Node, name: string): boolean {
   return getAttributeIndex(element, name) !== -1;
 }
 
+export function hasSpaceSeparatedAttributeValue(element: Node, name: string, value: string): boolean {
```

Comment:
If this function had the signature: `(name: string, value: string): Predicate` then you could use it with the other dom5 predicate APIs